style.scss

```diff
@@ -15,4 +15,9 @@ Text Domain: mojblocks
 @import 'node_modules/govuk-frontend/govuk/tools/px-to-rem';
 @import 'node_modules/govuk-frontend/govuk/settings/media-queries';
 @import 'node_modules/govuk-frontend/govuk/helpers/media-queries';
-@import 'src/**/style.scss';
\ No newline at end of file
+@import 'src/**/style.scss';
+
+/// hack: prevent a horizontal scrollbar in IE11
+main {
+    overflow-x: hidden;
+}
```

Comment:
I wonder if we should have a dedicated IE11 sass file or we just go with adding IE hacks in here...

Reply:
We should yes. I think we should revisit this conversation with the accessibility convo we need to have. What do you think?